src/utils/drawerWidth.ts

```diff
@@ -0,0 +1 @@
+export const drawerWidth = Math.min(640, window.innerWidth * 0.9)
```

Comment:
**suggestion**: utils are typically functions. WDYT about making this a function?